frontend/src/lib/components/ScriptBuilder.svelte

```diff
@@ -40,6 +42,7 @@
 
 	async function editScript(): Promise<void> {
 		try {
+			$dirtyStore = false
```

Comment:
you also want to initialize it on adding a new script

Reply:
This code is used to create and to update a script. `ScriptBuilder` component is used is both routes

Reply:
Yes, that's correct for saving but not fot initialiazing. At opening of a new script or editing an existing script, you want to initialize $dirtyStore to false also.

Reply:
Done